sync/migrate_plus.migration_group.default.yml

```diff
@@ -1,12 +0,0 @@
-uuid: d945a1f0-c82f-49dc-932c-0bddc63b19e1
-langcode: en
```

Comment:
This configuration file seems to no longer be necessary since upgrading the migrate_plus module.